src/main/java/duke/fileOperation/FileOperation.java

```diff
@@ -0,0 +1,85 @@
+package duke.fileOperation;
+
+
+import duke.tasks.*;
+
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import static duke.main.Duke.taskCount;
+
+public class FileOperation {
+    public static void initFile() throws IOException {
+        File newFile = new File("./taskslist.csv");
```

Comment:
I see you have referenced this value a couple times in this java file, consider assigning the value to variable then referencing it to minimise the use of magic literals.